.github/workflows/ci.js.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Node.js CI
```

Comment:
please get the habit correctly name your new pipelines JUST once;
CI, means anything related to the introduction of new code.
if you then would need to add a second pipeline for the CI process, it would be a problem;

(not blocking comment, but an habit to take);